### ISSUE TITLE
Investigate use_skip_connections

### ISSUE THREAD

**Issue by jkhlot** [philipperemy/keras-tcn#16]
I refer to  **TemporalBlock** in https://github.com/locuslab/TCN/blob/master/TCN/tcn.py
I found some slight difference in code  https://github.com/philipperemy/keras-tcn/blob/master/tcn/tcn.py.
```
Line 131:
 if use_skip_connections:
        x = keras.layers.add(skip_connections)
```

i think it should be:
`
 if use_skip_connections:
        x = keras.layers.add([skip_connections,x])
`
if i am wrong , pls correct me . Thanks

**Comment by philipperemy** [comment]
@jkhlot you might be right. I'll have to check at the code :)

**Comment by philipperemy** [comment]
@jkhlot noted! Going to have a look at those two points. Thanks for reporting :)

**Comment by philipperemy** [comment]
@jkhlot is there a particular why you closed this issue?

**Comment by philipperemy** [comment]
Regarding the wavenet activations,
- https://github.com/basveeling/wavenet/blob/bf8ef958372692ecb32e8540f7c81f69a186eb8d/wavenet.py#L235

Looks pretty similar to the one, no? tanh(x) * sigmoid(x)

**Comment by philipperemy** [comment]
Regarding the skip connections,
- https://arxiv.org/pdf/1609.03499.pdf
It does not seem that the final output x is merged with the skip connections. I might be wrong.
